build.sh

```diff
@@ -24,7 +24,7 @@ fi
 # Note that lsof returns a non-zero exit code if there is no match.
 SANDBOX_PID="$(lsof -ti tcp:6865 || true)"
 if [ -n "$SANDBOX_PID" ]; then
-    kill "$SANDBOX_PID"
+    echo $SANDBOX_PID | xargs kill
 fi
```

Comment:
You could potentially use `echo "$SANDBOX_PID" | xargs --no-run-if-empty kill` here instead.

Reply:
Ah, I didn't know about it thanks. 😃